tables/TaQL/TableGram.yy

```diff
@@ -408,17 +413,17 @@ withpart:  {   /* no WITH part */
 
 /* The SELECT command; note that many parts are optional which is handled
    in the rule of that part. The FROM part being optional is handled here
-   because later a join might be added. */
-selcomm:   withpart SELECT selcol FROM tables whexpr groupby having order limitoff given dminfo {
+   because a join might be used. */
+selcomm:   withpart SELECT selcol FROM tables joins whexpr groupby having order limitoff given dminfo {
                $$ = new TaQLQueryNode(
-                    new TaQLSelectNodeRep (*$3, *$1, *$5, 0, *$6, *$7, *$8,
-					   *$9, *$10, *$11, *$12));
```

Comment:
Sorry, I don't understand this code yet. I see that the fourth argument of `TaQLSelectNodeRep` is a `joins`. In the old code, that was `0`, probably meaning that joins were not supported. In the new code, the order of the arguments shift, and the `$6` that was the fifth argument in the old code, is now the fourth argument. Is the meaning of `$6` changed between the old code and the new code because extra stuff is parsed?